src/mo_frontend/typing.ml

```diff
@@ -1026,8 +1026,8 @@ let rec is_explicit_exp e =
     true
   | LitE l -> is_explicit_lit !l
   | UnE (_, _, e1) | OptE e1 | DoOptE e1
-  | ProjE (e1, _) | DotE (e1, _) | BangE e1 | IdxE (e1, _) | CallE (e1, _, _)
-  | LabelE (_, _, e1) | AsyncE (_, _, e1) | AwaitE (_, e1) ->
+  | ProjE (e1, _) | DotE (e1, _) | BangE e1 | IdxE (e1, _) | CallE (_(*FIXME: correct?*), e1, _, _)
```

Comment:
I think you can remove this. IIRC is_explicit_exp returns an indication whether the inferred type of the expression is good enough to resolve operator  overloading. But not sure. You'd have to use git blame to discover the meaning of this from Andrea's original PR that introduced is_explicit.

Reply:
4f4e93e03